File: examples/evidence-app/src/customer.ts
export const privacyActivity = "customer_registration";

export interface CreateCustomerDto {
  rut: string;
  email: string;
  fullName: string;
}

export async function registerCustomer(customer: CreateCustomerDto) {
  return fetch("/api/customers", {
    method: "POST",
    body: JSON.stringify({
      rut: customer.rut,
      email: customer.email,
      fullName: customer.fullName
    })
  });
}

File: examples/polyglot-app/go/customer.go
package main

import (
  "fmt"
  "net/http"
)

type Customer struct {
  Rut string
  Email string
}

func send(customer Customer) {
  fmt.Println(customer.Email)
  http.Post("https://api.example.com/customers", "application/json", nil)
}
